### ISSUE TITLE
New amazon developer gui

### ISSUE THREAD

**Issue by weha** [m0ngr31/kodi-alexa#264]
So I went ahead and tried to create the skill for alexa but the interface is renewed so the way that is explained here is no longer possible.
I was able to convert the intents with replace but the utterences need to go in per intent instead of all at once.
Would it be possible to create a json file (maybe amazon can convert a working one?) so I can use that in the new json editor?

thanks!

**Comment by m0ngr31** [comment]
I'll get the README updated to reflect this soon, but new docs that support the change are up [here](http://lexigr.am)

**Comment by weha** [comment]
@m0ngr31 

Yes I noticed the website on top after posting this but I can't get lexigram to work.
After poking around a bit it finally launched (having to disable service as well because port 9090 seems to be hardcoded) I accessed the webpage but it just keeps loading while saying: Switch to 'Login with Amazon' page...
After a while I get "connection refused"

Also, there's an error in the manual where it says N to continue instead of stopping but it is now Y apparently.
One of things that slowed me down because the commands after don't work if you follow it to the letter.
